virt/virt-2-4-release-notes.adoc

```diff
@@ -24,6 +24,17 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 //CNV-5438 - svirt-support-for-cnv
 * This release features significant security enhancements. {VirtProductName} now supports SELinux with Mandatory Access Control (MAC) for isolating virtual machines (VMs). Previously, all VMs were managed by using privileged xref:../security/container_security/security-platform.adoc#security-deployment-sccs_security-platform[Security Context Constraints (SCC)]. Now, you can use less privileged custom SCCs for VMs and limit the use of privileged SCCs to infrastructure containers in the cluster.
 
+//CNV-5472-Guest-OS
+* Supported guest operating system templates for {VirtProductName} include the following:
+** Windows 2012 R2, Windows 2016, Windows 2019, Windows 10.
+** RHEL 6, RHEL 7, RHEL 8.
```

Comment:
I think RHEL should come first in the list, and I think it should be written out like this:

```
** Red Hat Enterprise Linux 6, 7, and 8.
** Microsoft Windows Server 2012 R2, 2016, and 2019.
** Microsoft Windows 10.
```
(It seems like the Windows OSes should be split into server/not server.)

@fabiand let me know if you have different opinions :)

Reply:
Yes, let's follow this suggestion.

Reply:
Updated the content.